lms/djangoapps/verify_student/views.py

```diff
@@ -724,7 +731,7 @@ def create_order(request):
         return HttpResponseBadRequest(_("No selected price or selected price is below minimum."))
 
     if current_mode.sku:
```

Comment:
Just a side note on lines 675-702 above: it looks like the photo verification code is left over from before we launched the decoupled verified payment flow!  Definitely not required for this PR, but at some point it would be great to clean this up :)